### PR TITLE
(SIMP-7202) Add OEL8 support

### PR DESCRIPTION
**Pull request by trevor-vaughan**

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
⏳ https://gitlab.com/simp/pupmod-simp-auditd/pipelines/103512248

File: spec/acceptance/suites/default/lib/util.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          end
+        end
+
+        tempname = '/tmp/auditd-' + SecureRandom.uuid+ '.rules'
+
+        create_remote_file(host, tempname, ['-c', @rules].flatten.join("\n"))
+        on(host, "chmod 600 #{tempname}")
+
+        auditctl_output = on(host,
+          "auditctl -R #{tempname}",
```
Comment (jeannegreulich) [inline spec/acceptance/suites/default/lib/util.rb]: Auditctl doesn't do the same checking that augenrules --load does.  It never recognizes that files or users aren't there so if all you are going to do is an auditctl  to check the rules do you need all this fancy stuff?

**Comment by trevor-vaughan** [comment]
@jeannegreulich `augenrules` calls `auditctl` under the hood but then gives a bunch of other information that isn't relevant.

The regexes remove rules that we know might not work (and that's OK) and then checks the rest for errors or warnings.

This allows us to catch issues as they arise in the future.

File: spec/acceptance/suites/default/lib/util.rb (hunk 1 above)
Comment (trevor-vaughan) [reply spec/acceptance/suites/default/lib/util.rb]: If you remove the regexes and run it, you can watch it fail correctly.

**Comment by jeannegreulich** [comment]
If you put entries in a audit.rules files that have files that don't exist or uids that don't exist and run auditctl -R on that file you get no errors.  It does not check for that stuff.  augenrules must check it somewhere else.    It only seems to error out if there is an error in the rule.  So the removal fo entries in the audit.rules file does not seem necessary.

**Pull request merged**